jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -377,6 +377,27 @@ def add_data(self, data, color=None, alpha=None, **layer_state):
             if "Subset" in layer.layer.label and layer.layer.data.label == data.label:
                 layer.linewidth = 3
 
+        # This default color cycle was retrieved from matplotlib with the following lines:
+        #   import matplotlib.pyplot as plt
+        #   plt.rcParams['axes.prop_cycle'].by_key()['color']
+
+        color_cycle = [
```

Comment:
Glue or matplotlib must already have one that we can use?

Reply:
Similar thoughts here: https://github.com/spacetelescope/jdaviz/pull/1674#issuecomment-1261130604